.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.8
```

Comment:
Any 2.7 version should work

Reply:
I tried to remove the path versioning but it seems that exact versioning needs to be there. I added "this is the latest '2' major version release as per requirements" for 2.7, if not let me know which path version we can use